packages/data-point/src/reducer-helpers/assign/assign.test.js

```diff
@@ -0,0 +1,57 @@
+const { ReducerAssign } = require("./assign");
+const { Accumulator } = require("../../Accumulator");
+const { resolve } = require("../../resolve");
+
+describe("ReducerAssign", () => {
+  describe("constructor", () => {
+    it("should set type to 'constant'", () => {
+      const reducer = new ReducerAssign({});
+      expect(reducer.type).toEqual("assign");
+    });
+
+    it("should create a reducer from spec and assign to assignReducer", () => {
+      const reducer = new ReducerAssign({});
+      expect(reducer.assignReducer).toHaveProperty("type", "object");
```

Comment:
return assignReducer to just reducer? since that function itself doesn't call assign

Reply:
there are cases where more than one variable could be set, and not all could be reducers, for example the `select` and the `ifThenElse`. I do like the idea, would suggest we do a revision of this idea once we have all the helpers merged and we can see if we can add some naming pattern for these types of class properties.